app/views/govuk_publishing_components/components/docs/password_input.yml

```diff
@@ -0,0 +1,42 @@
+name: Password input
+description: The password input component helps users to create and enter passwords.
+govuk_frontend_components:
+  - password-input
+body: |
+  This component is based on the [GOV.UK Design System password input component](https://design-system.service.gov.uk/components/password-input/)
+
+accessibility_criteria: |
+  The password input component must:
+
+    * accept focus
+    * be focusable with a keyboard
+    * be usable with a keyboard
+    * be usable with touch
+    * indicate when they have focus
+    * be recognisable as form input elements
+    * have correctly associated labels
+    * have the appropriate type of 'password'
+    * have an appropriate validation message if there was an error
+    * allow users to interact with any 'show password' button
+    * allow users to use `autocomplete` to securely create and enter passwords
+    * allow users to copy and paste into the password input
+
+  When CSS and / or JavaScript is unavailable, the component must:
+
+  * fallback to a regular input element, with the show/hide button hidden
+
+uses_component_wrapper_helper: true
+
+examples:
+  default:
+    data:
+  with_error:
+    description: If there is a validation error, passing error text will style the password input component with error styles, and semantically state that there was a validation error.
+    data:
+      error_text: "Enter a password"
+  with_custom_text:
```

Comment:
Could this be `with_custom_label_and_text`? I was initially confused reading it.